stable/pipelines/values.yaml

```diff
@@ -502,7 +502,7 @@ pipelines:
         defaultExchange: {{ .Values.rabbitmq.root_vhost_exchange_name }}
         amqpVhost: {{ .Values.rabbitmq.build_vhost_name}}
         amqpRootVhost: {{ .Values.rabbitmq.root_vhost_name }}
-        protocol: amqps
+        protocol: amqp
```

Comment:
Please an option to set `amqp` or `amqps`

Reply:
I agree this should not be a default, there should be an option to set `amqp` or `amqps`

Reply:
Please make it as similar to Xray as possible. We would like to eventually share config.
Also, it might be a good idea to add support for the `global` option here... See https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#global-chart-values